oasis-reference/src/main/kotlin/com/empowerops/volition/ref_oasis/OptimizationControl.kt

```diff
@@ -0,0 +1,14 @@
+package com.empowerops.volition.ref_oasis
+
+interface OptimizationControl {
+    suspend fun startOptimization(optimizer: Optimizer)
+    suspend fun stopOptimization(): Boolean
+    suspend fun cancel(name: String)
+    suspend fun cancelAll()
+    suspend fun cancelAndStop()
+    suspend fun updateNode(name: String)
+    suspend fun syncAll()
+
+    fun disconnectAll()//TODO
+    fun unregisterAll()//TODO
+}
```

Comment:
So `OptimizationControl ` is an interface that is effectively our... well what is it? Its the API that the UI can program against? But no user is going to want it, and we're never going to have any other implementations of it? What problem does it solve? Is there some dependency your looking to invert that I'm not understanding?

Reply:
it wasn't well named either. yeah, as we speak, there isn't much reason for now so I will remove it. It is all me try to get some level of abstraction when I looking at a class have too many roles.